### DISCUSSION TITLE
Switching Java Version not working on Synology NAS

### DISCUSSION THREAD

**Discussion by TimonH-98** [itzg/docker-minecraft-server#2324] [General]
Hello,
Iam trying to switch the JAVA Version on for a 1.12.2 Forge Server. 
That are my settings (not working):

![Screenshot 2023-08-07 135018](https://github.com/itzg/docker-minecraft-server/assets/89013503/7a2b615b-957d-4e13-a6b6-e03d9e62fbe6)

What should i do to get it running? My Server is crashing because its using the newest Java Version and not java8.
I also have tryed tipping in "java8" "java8-multiarch" and similar stuff

Thanks for the help!

**Comment by itzg** [comment]
You will need to use one of the Java 8 image tags rather than changing env vars 

https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/#running-minecraft-server-on-different-java-version

**Reply by TimonH-98** [reply]
And how do i do that on Synology? Iam very confused

**Reply by itzg** [reply]
I'm personally not sure. Looking at these docs https://kb.synology.com/en-nz/DSM/help/Docker/docker_container?version=6 I can't see where image tags are selected. Maybe in that "Settings" button in the Registry section?

**Reply by TimonH-98** [reply]
nope. But i can import/export settings from a container: 

{
   "CapAdd" : null,
   "CapDrop" : null,
   "cmd" : "",
   "cpu_priority" : 90,
   "enable_publish_all_ports" : false,
   "enable_restart_policy" : false,
   "enable_service_portal" : null,
   "enabled" : false,
   "env_variables" : [
      {
         "key" : "PATH",
         "value" : "/opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
      },
      {
         "key" : "JAVA_HOME",
         "value" : "/opt/java/openjdk"
      },
      {
         "key" : "LANG",
         "value" : "en_US.UTF-8"
      },
      {
         "key" : "LANGUAGE",
         "value" : "en_US:en"
      },
      {
         "key" : "LC_ALL",
         "value" : "en_US.UTF-8"
      },
      {
         "key" : "JAVA_VERSION",
         "value" : "java8"
      },
      {
         "key" : "TYPE",
         "value" : "Forge"
      },
      {
         "key" : "VERSION",
         "value" : "1.12.2"
      },
      {
         "key" : "EULA",
         "value" : "true"
      },
      {
         "key" : "UID",
         "value" : "1000"
      },
      {
         "key" : "GID",
         "value" : "1000"
      },
      {
         "key" : "MEMORY",
         "value" : "8G"
      }
   ],
   "exporting" : false,
   "id" : "19208d81f97a3ce2cd2c884856b331240b80bde08fc2a8a43e7f4c602a99f0bd",
   "image" : "itzg/minecraft-server:latest",
   "is_ddsm" : false,
   "is_package" : false,
   "links" : [],
   "memory_limit" : 9407823872,
   "name" : "TEST_1.12.2",
   "network" : [
      {
         "driver" : "bridge",
         "name" : "bridge"
      }
   ],
   "network_mode" : "bridge",
   "port_bindings" : [
      {
         "container_port" : 25565,
         "host_port" : 25569,
         "type" : "tcp"
      }
   ],
   "privileged" : false,
   "shortcut" : {
      "enable_shortcut" : false,
      "enable_status_page" : false,
      "enable_web_page" : false,
      "web_page_url" : ""
   },
   "use_host_network" : false,
   "volume_bindings" : [
      {
         "host_volume_file" : "/docker/Minecraft/Test1.12.2",
         "mount_point" : "/data",
         "type" : "rw"
      }
   ]
}

**Reply by itzg** [reply]
I guess change the `"image"` there then.

**Reply by TimonH-98** [reply]
I tryed that before. While importing i get the error that this image cant be found.

**Reply by TimonH-98** [reply]
ohhhh wait i found it! While downloading the image i can change the version and there are sooo many versions to download. I just need to find the java8 one
Thanks!
